guides/common/modules/proc_configuring-dynamic-dns-update-with-tsig-authentication.adoc

```diff
@@ -67,6 +67,12 @@ grant "rndc-key" zonesub ANY;
 # chmod -v 640 /etc/rndc.key
 ----
 
+. Normally, {foreman-installer} makes sure the `foreman-proxy` user belongs to `named` UNIX group, but in this specific configuration scenario management of the service is disabled therefore the group will not be assigned. Assign the user to the correct group manually:
++
+[options="nowrap"]
+----
+# usermod -a -G named foreman-proxy
```

Comment:
I've always used `gpasswd -a foreman-proxy named`. I started to use this because you can also use it to locally add a user to a group when the isn't managed locally, for example when it's in LDAP. Probably not an issue, but something that jumped out to me.